src/murfey/server/ispyb.py

```diff
@@ -58,9 +90,30 @@ def get_all_ongoing_visits(microscope: str, db: sqlalchemy.orm.Session) -> list[
         Visit(
             start=row.startDate,
             end=row.endDate,
+            session_id=row.sessionId,
             name=f"{row.proposalCode}{row.proposalNumber}-{row.visit_number}",
             proposal_title=row.title,
             beamline=microscope,
         )
         for row in query
     ]
+
+
+def start_data_collection(db: sqlalchemy.orm.Session):
+    comment = "Test Murfey DC insert"
+    insert = _DataCollection(comments=comment)
+    db.add(insert)
+    db.commit()
+
+
+def get_data_collection_group_ids(session_id):
+    query = (
+        Session()
+        .query(_DataCollectionGroup)
+        .filter(
+            _DataCollectionGroup.sessionId == session_id,
+        )
+        .all()
+    )
+    dcgids = [row.dataCollectionGroupId for row in query]
+    return dcgids
```

Comment:
Note: This is wasteful. The query retrieves every field and throws all of them (but one) away.

(However, the approach to inserting the DCID is more fundamentally broken so this doesn't matter much)